docs/range-tokens/usage-tutorial.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Range Tokens Usage Tutorial
+sidebar_label: Usage Tutorial
+---
+
+Now that you have a deployed LSP contract, this tutorial will walk you through the four ways to interact with the contract and your range tokens.
```

Comment:
I think this page is good, but might be more understandable with an actual concrete example (similar recommendation as the one for the deployment tutorial). It'd be cool to use the same contract example for both the deployment and usage tutorial

Reply:
Included examples using the UMA range token. Let me know if there is anywhere you see an opportunity to add an example that I missed.